packages/beacon-node/src/api/impl/lodestar/index.ts

```diff
@@ -5,7 +5,7 @@ import {routes} from "@lodestar/api";
 import {ApplicationMethods} from "@lodestar/api/server";
 import {ChainForkConfig} from "@lodestar/config";
 import {Repository} from "@lodestar/db";
-import {ForkName, ForkSeq, SLOTS_PER_EPOCH} from "@lodestar/params";
```

Comment:
I observed this case but but this can be avoided manually for now running "Organize Imports" command manually in VSCode.

or would be nice to add following in the VSCode settings in this PR. This will make sure to have no unused imports for the files someone edit in VSCode.

```json
  "editor.codeActionsOnSave": {
    "source.organizeImports": "always",
  }
```

Reply:
> or would be nice to add following in the VSCode settings in this PR. This will make sure to have no unused imports for the files someone edit in VSCode.

we already have this, isn't it the same?

https://github.com/ChainSafe/lodestar/blob/27301c36341c7df70fad349993da7116ec4e7115/.vscode/settings.json#L8

Reply:
you are right that `"source.organizeImports": "always"` removed unused imports but I noticed there is some conflict in formatting, any time you safe the file the imports jump around strangely, I would rather keep this setting out for now

the `"noUnusedLocals": true` has a lot of other nice benefits, I guess would be nice to not have to worry about imports manually but you get a yellow warning in vs code, so very likely caught before pushing the code

Reply:
this also sorts the imports differently and causes `yarn lint` to fail, it's basically vs code fighting against biome